sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_storage.py

```diff
@@ -182,9 +187,36 @@ def get(self):
         return None
 
     def put(self, data, lease_period=None):
-        if not self._enabled:
+        # Create path if it doesn't exist
+        metrics = self._customer_statsbeat_metrics
+        try:
```

Comment:
I don't believe you need a try: catch here. Can simply just record the metric if `put` is called and `self._enabled` is false.